x-pack/platform/plugins/shared/embeddable_alerts_table/public/actions/add_alerts_table_action.ts

```diff
@@ -41,17 +40,28 @@ export const getAddAlertsTableAction = (
     },
     execute: async ({ embeddable }) => {
       if (!apiIsPresentationContainer(embeddable)) throw new IncompatibleActionError();
-      const tableConfig = await openConfigEditor({
-        coreServices,
+
+      openLazyFlyout({
+        core: coreServices,
         parentApi: embeddable,
-      });
-      await embeddable.addNewPanel(
-        {
-          panelType: EMBEDDABLE_ALERTS_TABLE_ID,
-          serializedState: { rawState: { tableConfig } },
+        loadContent: async ({ closeFlyout, ariaLabelledBy }) => {
+          const { ConfigEditor } = await import('../components/config_editor');
+          return ConfigEditor({
```

Comment:
nit: can we convert this to a tsx file and use a JSX expression?

```suggestion
          return <ConfigEditor
```